packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -313,12 +313,25 @@ export default class RunPanelStore {
     @action.bound
     showErrorMessage(data) {
         const { journal } = this.root_store;
+
         journal.onError(data);
         if (journal.journal_filters.some(filter => filter === message_types.ERROR)) {
             this.setActiveTabIndex(2);
+        } else {
+            this.root_store.ui.addNotificationMessage(journalError(this.switchToJournal));
         }
     }
 
+    @action.bound
+    switchToJournal = () => {
```

Comment:
Nit, can be a class function rather than arrow function so we stay consistent across the file.

```suggestion
    switchToJournal() {
```